### DISCUSSION TITLE
Welcome to todo-list Discussions!

### DISCUSSION THREAD

**Discussion by benjamin-codespace** [benjamin-codespace/todo-list#7] [Announcements]
Hi Rocio, maybe we can talk on here.

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by benjamin-codespace** [comment]
Test?!

**Reply by RocioPG3** [reply]
OK hablamos por aquí!

**Reply by benjamin-codespace** [reply]
Seguimos mañana?

**Reply by RocioPG3** [reply]
Buenos días! ya he hecho mi parte, no sé como hacer el subrayado en el punto de "Installing" 
Give the example y until finished

No sé si me puedes ayudar sino esta tarde lo vemos con Antonio

¡Un saludo!

**Reply by benjamin-codespace** [reply]
Buenos dìas, intentalo con un blockquote >

**Reply by RocioPG3** [reply]
Genial, gracias actualizado :)

**Reply by RocioPG3** [reply]
Creo que nosotros ya hemos terminado
